app/Nova/Resources/Wiki/ExternalResource.php

```diff
@@ -182,6 +183,24 @@ public function fields(Request $request): array
                     ];
                 }),
 
+            BelongsToMany::make(__('nova.studio'), 'Studio', Studio::class)
```

Comment:
Acceptance test finding: 'Studio' needs to be 'Studios'. This throws an exception when viewing an external resource in nova.